gpflow/conditionals.py

```diff
@@ -171,55 +171,62 @@ def base_conditional(Kmn, Kmm, Knn, f, *, full_cov=False, q_sqrt=None, white=Fal
       q(g2) = N(g2;f,q_sqrt*q_sqrt^T)
     This method computes the mean and (co)variance of
       q(g1) = \int q(g2) p(g1|g2)
-    :param Kmn: M x N
-    :param Kmm: M x M
-    :param Knn: N x N  or  N
+    :param Kmn: ... x M x N
+    :param Kmm: ... x M x M
+    :param Knn: ... x N x N  or ... x N
     :param f: M x R
     :param full_cov: bool
     :param q_sqrt: None or R x M x M (lower triangular)
     :param white: bool
-    :return: N x R  or R x N x N
+    :return: ... x N x R  or ... x R x N x N
```

Comment:
It became hard to read when spaces are used. What do you think about `... x N x R` => `(...xNxR)`?

Reply:
@awav With "x" the letter separating the dimension sizes, I personally find it harder to read when spaces are NOT used ... maybe use the multiplication symbol "×" instead? As in N×N×R ?

Reply:
I personally find [N, D] easier to read, but happy with whatever